code/modules/projectiles/guns/energy/laser.dm

```diff
@@ -161,3 +161,4 @@
 	needs_permit = 0
 	ammo_x_offset = 2
 	selfcharge = 1
+	

```

Comment:
What a weird place to add a new line haha

Reply:
yeah i had a pulse laser tag gun for the CC costume that's been removed, didn't handle that file properly and am not sure how to fix it now lmao

Reply:
Removing the line from the file and committing should fix it, but no sweat if you don't 😄